central/main.go

```diff
@@ -703,15 +703,15 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 		{
 			Route:      "/db/backup",
 			Authorizer: dbAuthz.DBReadAccessAuthorizer(),
-			ServerHandler: notImplementedOnManagedServices(
+			ServerHandler: notImplementedWithExternalDatabase(
```

Comment:
perhaps not for this PR, but is there a way to get the secrets through the backups only? That may still be useful. something to think about